.lintr

```diff
@@ -2,7 +2,8 @@ linters: linters_with_defaults(
 	line_length_linter = NULL,
 	commented_code_linter = NULL,
 	indentation_linter = indentation_linter(4L),
-	object_name_linter = object_name_linter(styles = c("snake_case", "symbols", "SNAKE_CASE"))
+	object_name_linter = object_name_linter(styles = c("snake_case", "symbols", "SNAKE_CASE")),
+	object_usage_linter = NULL
```

Comment:
Is there a reason to remove this lint ?

Reply:
Yes. It sucks. You can see on the CI run for the previous commit that it could not understand that `ui` and `server` were defined in another file, and flagged them as a warning. There's a workaround but having to work around the linter is completely ridiculous, so IMO it's better to just disable it. We'll know right away if we're using undefined objects anyway.

Reply:
Ok ok